clang/test/Sema/format-strings-cheri.c

```diff
@@ -15,9 +19,12 @@ void test_long_pointer(void * __capability p) {
   printf("%.4lp", p); // expected-warning{{precision used with 'p' conversion specifier, resulting in undefined behavior}}
   printf("%#.4lp", p);
   printf("%lp", &p); // hybrid-warning{{format specifies type 'void * __capability' but the argument has type 'void * __capability *'}}
```

Comment:
Maybe put these on a newline too now you have purecap-pedantic-warning versions?

Reply:
sounds good, will do that once Jenkins completes.